flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -1163,7 +1184,7 @@ static void setJobManagerAddressInConfig(Configuration config, InetSocketAddress
 	 * @param className The fully-qualified class name to load.
 	 * @param params The constructor parameters
 	 */
-	private static CustomCommandLine<?> loadCustomCommandLine(String className, Object... params) throws IllegalAccessException, InvocationTargetException, InstantiationException, ClassNotFoundException, NoSuchMethodException {
+	private static CustomCommandLine loadCustomCommandLine(String className, Object... params) throws IllegalAccessException, InvocationTargetException, InstantiationException, ClassNotFoundException, NoSuchMethodException {
```

Comment:
We catch `Exception` in the only caller. I think it make sense we declare this method `throws Exception` since we don't care the detailed type.